src/components/mobile-offer-header/MobileOfferHeader.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="mobile-offer-header">
+    <span>{{ text.toUpperCase() }}</span>
+    <div class="wrapper">
+      <p>{{ userName.charAt(0).toUpperCase() + userName.slice(1) }} &nbsp;</p>
```

Comment:
Will this work as intended if you pass an empty string as username?

Reply:
It will not display anything